stdlib/public/TensorFlow/Tensor.swift

```diff
@@ -536,10 +536,12 @@ public extension Tensor where Scalar : Numeric {
   @_inlineable @inline(__always)
   init(oneHotAtIndices indices: Tensor<Int32>, depth: Int32,
        onValue: Scalar = 1, offValue: Scalar = 0, axis: Int = -1) {
-    self.init(
-      handle: #tfop("OneHot", indices, Tensor<Int32>(depth),
-                    Tensor(onValue), Tensor(offValue),
-                    axis: axis, T: Scalar.self, TI: Int32.self)
+    self = Raw.oneHot(
+      indices: indices,
+      depth: Tensor<Int32>(depth),
```

Comment:
The `Int32` generic signature can be omitted.

Reply:
Not sure about this, when trying it I got:
```
Tensor.swift:541:14: error: type of expression is ambiguous without more context
      depth: Tensor(depth),
             ^~~~~~~~~~~~~
```
Maybe Tensor without the generics bit means Tensor<Scalar> in this context.